documentation/README.ex.md

```diff
@@ -0,0 +1,34 @@
+# Repository title
+
+> Provides an example readme template to bootstrap documentation.
+
+Start with one sentence explaining what the repository does. Avoid mentioning the technology used, its rationale, or tech jargon. This sentence should be understandable by a non-technical audience.
+
+## Development
+
+Describe the development environment. Include basic instructions--does this repository prefer `npm ci` or `npm i`? Does prettier run on save? Any special settings or extensions you should set up in your IDE?
+
+If you're not sure where to start, consider explaining the npm scripts that might be relevent for getting a development environment
```

Comment:
Probably here we should add the environment variables needed to run it or to modify its behaviour.

Reply:
Good call, thank you!